packages/core/saved-objects/core-saved-objects-migration-server-internal/src/document_migrator/model_version.ts

```diff
@@ -32,45 +34,54 @@ export const getModelVersionTransforms = ({
       ? typeDefinition.modelVersions()
       : typeDefinition.modelVersions ?? {};
 
-  const transforms = Object.entries(modelVersionMap)
-    .filter(([_, definition]) => !!definition.modelChange.transformation)
-    .map<Transform>(([rawModelVersion, definition]) => {
-      const modelVersion = assertValidModelVersion(rawModelVersion);
-      const virtualVersion = modelVersionToVirtualVersion(modelVersion);
-      return {
-        version: virtualVersion,
-        transform: convertModelVersionTransformFn({
-          log,
-          modelVersion,
-          virtualVersion,
-          definition,
-        }),
-        transformType: TransformType.Migrate,
-      };
-    });
-
-  return transforms;
+  return Object.entries(modelVersionMap).map<Transform>(([rawModelVersion, definition]) => {
+    const modelVersion = assertValidModelVersion(rawModelVersion);
+    const virtualVersion = modelVersionToVirtualVersion(modelVersion);
+    return {
+      version: virtualVersion,
+      transform: convertModelVersionTransformFn({
+        log,
+        modelVersion,
+        virtualVersion,
+        definition,
+        type: 'up',
+      }),
+      transformDown: convertModelVersionTransformFn({
+        log,
+        modelVersion,
+        virtualVersion,
+        definition,
+        type: 'down',
+      }),
+      transformType: TransformType.Migrate,
+    };
+  });
 };
 
 export const convertModelVersionTransformFn = ({
   virtualVersion,
   modelVersion,
   definition,
+  type,
   log,
 }: {
   virtualVersion: string;
   modelVersion: number;
   definition: SavedObjectsModelVersion;
+  type: 'up' | 'down';
   log: Logger;
 }): TransformFn => {
   if (!definition.modelChange.transformation) {
-    throw new Error('cannot convert model change without a transform');
+    return noopTransform;
   }
```

Comment:
I had to change the behavior and create noop transformations for model versions not having a `transformation`. The reason is that otherwise, the pipeline would not know about the versions not having transformations, which can cause problems if the latest version is in this case (and therefor the `lastVersion` know of the pipeline is lower than the actual version of the docs).